src/Tile_Maps.md

```diff
@@ -33,15 +33,39 @@ Bit 3    Tile VRAM Bank number      (0=Bank 0, 1=Bank 1)
 Bit 2-0  Background Palette number  (BGP0-7)
 ```
 
-When Bit 7 is set, the corresponding BG tile will have priority above
-all OBJs (regardless of the priority bits in OAM memory). There's also
-a Master Priority flag in LCDC register Bit 0 which overrides all other
-priority bits when cleared.
-
 Note that, if the map entry at `0:9800` is tile \$2A, the attribute at
 `1:9800` doesn't define properties for ALL tiles \$2A on-screen, but only
 the one at `0:9800`!
 
+### BG-to-OBJ Priority in CGB Mode
+
+In the CGB mode there are 3 different places where the priority between the BG (and window) layer and the OBJ layer is declared:
+1. [BG Map Attribute bit 7](#bg-map-attributes-cgb-mode-only) - BG-to-OAM Priority (CGB Mode)
+2. [LCDC bit 0](LCDC.md#lcdc0--bg-and-window-enablepriority) - BG and Window Priority (CGB Mode)
+3. [OAM Attributes bit 7](OAM.md#byte-3--attributesflags) - BG and Window over OBJ
+
+
+The following table visualize the relations between the 3 flags
+> **Notice:** OAM Attributes bit 7 will grant OBJ priority when clear and not when set
+
+| OAM Attributes bit 7 | BG Map Attribute bit 7 | LCDC bit 0 | Priority |
+| -------------------- | ---------------------- | ---------- | -------- |
+| [_] | [X] | [X] | BG color 0 - OBJ, otherwise BG |
+| [_] | [X] | [_] | OBJ |
+| [X] | [_] | [X] | BG color 0 - OBJ, otherwise BG |
+| [X] | [X] | [X] | BG color 0 - OBJ, otherwise BG |
+| [_] | [_] | [X] | OBJ |
+| [_] | [_] | [_] | OBJ |
+| [X] | [X] | [_] | OBJ |
+| [X] | [_] | [_] | OBJ |
```

Comment:
Same here, I think this kind of "condition" table could be rendered in another way/more compliant to the rest of the document, especially because "[_]" and "[X]" are very unusual ways to render a Yes/No 1/0

Reply:
Replaced with 1/0